src/components/cc-tile-status-codes/cc-tile-status-codes.js

```diff
@@ -89,25 +113,27 @@ export class CcTileStatusCodes extends LitElement {
             onClick: function (e, legendItem) {
               this.chart.data.labels.forEach((label, i) => {
                 const sameLabel = label === legendItem.text;
+                // @ts-expect-error shiftKey may be undefined but it won't trigger any issue anyway
                 if (xor(e.native.shiftKey, sameLabel)) {
                   this.chart.toggleDataVisibility(i);
                 }
               });
               this.chart.update();
             },
-            onHover: (e) => {
-              this._ctx.style.cursor = 'pointer';
+            onHover: () => {
+              this._refCanvas.value.style.cursor = 'pointer';
             },
-            onLeave: (e) => {
-              this._ctx.style.cursor = null;
+            onLeave: () => {
+              this._refCanvas.value.style.cursor = null;
             },
             position: 'right',
             labels: {
-              fontFamily: 'monospace',
+              font: { size: 16, family: 'monospace' },
               usePointStyle: true,
               // Filter legend items so we can only keep 1xx, 2xx... instead of all status codes
               filter: (current, all) => {
                 const label = current.text;
+                // @ts-expect-error TODO: find a solution
```

Comment:
suggestion:

```js
/**
 * @typedef {import('chart.js').LegendItem} LegendItem
 */


              /**
               * @param {LegendItem & {index: number}} current
               * @param all
               * @return {boolean}
               */
              filter: (current, all) => {
                const label = current.text;
                const previousLabel = all.labels[current.index - 1];
                return label !== previousLabel;
              },
```

Reply:
Actually this is an issue with chartjs itself, upgrading it makes the error go away, I just forgot to change the explanation :innocent: (I plan to upgrade chartjs in a separate PR soon)